### PR TITLE
docs: Update `<Aspect>` component page

### PR DESCRIPTION
**Pull request by sebald**
# Description

Revised the whole page with more awesome information and usage examples

# Reviewers:

@marigold-ui/developer
@marigold-ui/designer

### CODE REVIEW

File: docs/content/components/layout/aspect/aspect.mdx [hunk 1]
```diff
@@ -1,42 +1,78 @@
...
-The `<Aspect>` layout component dynamically modifies the height of an element based on its current width to match a width-to-height ratio. For example, a 4:3 ratio would make an element with a width of `400px` have a height of `300px`. This updates in real time when you resize the screen.
+The `<Aspect>` component keeps media elements like images or videos in their correct aspect ratio, preventing them from looking stretched or squished on different screens. It is particularly useful for creating responsive layouts where consistent media dimensions are important.
 
-## Import
+## Usage
 
-```tsx
-import { Aspect } from '@marigold/components';
-```
+When using the "aspect" component to maintain the aspect ratio of media elements, choose an appropriate predefined ratio to ensure a consistent and visually appealing layout. The available aspect ratios include:
```
Comment (aromko) [inline docs/content/components/layout/aspect/aspect.mdx:11]: `<Aspect>`instead of "aspect"

Comment (sebald) [reply docs/content/components/layout/aspect/aspect.mdx:11]: whoops

**Review by aromko — DISMISSED** [review]

**Comment by sebald** [comment]
@aromko There is a test failing, but I can not reproduce it locally :-/

**Comment by sebald** [comment]
@aromko @sarahgm solved it! 🥳

**Review by aromko — APPROVED** [review]

**Pull request merged**